src/bin.js

```diff
@@ -46,16 +46,10 @@ commander
 
 // Number List
 commander
-  .command('numbers:list')
-  .description('List of numbers assigned to the account')
-  .option('--page <page>', 'the page of results to return', /^\d*$/i, 1)
-  .option('--size <size>', 'the amount of results to return', /^\d*$/i, 100)
-  .alias('nl')
-  .action(request.numbersList.bind(request));
-
-commander
   .command('number:list', null, { noHelp: true })
   .alias('numbers')
+  .alias('numbers:list')
```

Comment:
That seems to confirm what I remembered of it. We should probably fix the number:buy one